app/Views/ScBooking/AvailableTimes.cshtml

```diff
@@ -24,7 +24,7 @@
         <h6>Available Times in @Model.BookingLocationName</h6>
         <p>Please choose a time below for your <b>@Model.HearingTypeName</b>.
         <br/>
-            @ScHearingType.GetHearingType(Model.HearingTypeId)s are booked for
+            @ScHearingType.GetHearingType(Model.HearingTypeId)’s are booked for
```

Comment:
I think the code was actually correct in this case. An apostrophe indicates possession, not plural.

Reply:
I'll revert it 👍 I just copied it from Figma and I do sometimes see it for "abbreviations" (but I wouldn't personally put one there either)